packages/dappmanager/hostScripts/docker_update.sh

```diff
@@ -0,0 +1,343 @@
+#!/bin/bash
+
+###################
+##### OPTIONS #####
+###################
+
+# engine
+#    -v | --version : returns string with docker-server version
+#    -i | --install : installs docker engine using "package method". If error returns string error
+# compose
+#    -v | --version : returns string with docker compose version
+#    -i | --install : installs docker compose. If error returns string error
+# system: returns system info: OS, version and docker versions (compose and engine)
+
+#####################
+### CONFIGURATION ###
+#####################
+
+# non-case sensitive (only available in bash)
+shopt -s nocasematch
+
+# Exit on error
+set -e
+
+####################
+### ERRORS CODES ###
+####################
+
+# 2 => no match requirements
+# 3 => unable to retrieve architecture
+# 4 => unable to retrieve os info
+# 5 => docker does not exist
+# 6 => docker compose does not exist
+# 7 => docker engine is stable
+# 8 => docker compose is stable
+# 9 => architecture not supported
+# 10 => Debian version not supported by docker and/or dappnode
+# 12 => Only 1 argument accepted
+# 13 => Arg must be docker-engine or docker-compose
+# 14 => flag must be -i | --install OR -v | --version
+# 15 => Illegal to downgrade docker engine (current docker version > than stable docker version)
+# 16 => Illegal to downgrade docker compose (current compose version > than stable compose version)
+
+###################
+#####VARIABLES#####
+###################
+
+# Docker stable versions
+STABLE_DOCKER_ENGINE_VERSION="20.10.2" # Same for PKG and CLI
+STABLE_DOCKER_CONTAINERD_VERSION="1.4.3-1"
+
+# Docker compose stable versions
+STABLE_DOCKER_COMPOSE_VERSION="1.25.5"
+
+# Download
+WGET="wget -q -O"
```

Comment:
Not a fan of this alias, adds complexity in exchange to make the commands a tinny bit shorter